diff --git a/node_modules/tmi.js/index.js b/node_modules/tmi.js/index.js
index 04a2a89..cd277f0 100755
--- a/node_modules/tmi.js/index.js
+++ b/node_modules/tmi.js/index.js
@@ -1,5 +1,9 @@
-const client = require('./lib/client');
-module.exports = {
-	client,
-	Client: client
-};
+// Your own custom client module
+import { Client } from 'tmi.js';
+
+// You can create a custom client instance if you need to
+const client = new Client();
+
+// Then export what you need using ES Module syntax
+export { client };
+export { Client };
